docs/_articles/en/lwc/write-lwc.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Write Lightning Web Components in VS Code
+lang: en
+---
+
+The Lightning Web Components Extension for VS Code leverages the editor's built-in code navigation and language features so that you can efficiently build Lightning web components. 
+
+Take advantage of the following Lightning Web Components Extension for VS Code features:
+- [Code Completion](#code-completion)
+- [View Documentation on Hover](#view-documentation-on-hover)
+- [View Errors Messages on Hover](#view-error-messages-on-hover)
+- [Linting](#linting)
+
+## Code Completion
+
+The Lightning Web Components Extension builds on VS Code's code completion capabilities by completing resource names in HTML and JavaScript files.
+
+VS Code completes static resources. In this example, it completes the `chartjs` object in the `libChartjs` component. `libChartjs` refers to the third-party JavaScript library [chartjs](https://www.chartjs.org/). Include it in your code by importing `loadScript` from the Lightning Web Components `platformResourceLoader` module. Then, add the statement `import chartjs from '@salesforce/resourceUrl/chart';`. Here, VS Code fills in the resource's name when we add it in the import statement.
```

Comment:
I think the examples a little need work. We probably need to decide what examples to show vs which ones to just mention. If you are using the lwc-recipes project, example completions you could screenshot are:

Apex:
@salesforce/apex/AccountController.getSingleAccount

Content assets:
@salesforce/contentAssetUrl/recipes_sq_logo

Static Resources:
@salesforce/resourceUrl/trailhead_logo

Schema / User:
@salesforce/schema
@salesforce/user

Lightning APIs (lists / records / etc):
lightning/*

LWC syntax completions (not import statements):
@wire
@api
@track
etc